rudder-core/src/main/scala/com/normation/rudder/domain/reports/ExpectedReports.scala

```diff
@@ -235,7 +235,7 @@ object ExpectedReportsSerialisation {
                ~ ("components"  -> (d.components.map { c =>
                    (
                      ("componentName" -> c.componentName)
-                   ~ ("cardinality"   -> c.cardinality)
+                   // ~ ("cardinality"   -> c.cardinality // #10625: ignore cardinality)
```

Comment:
I think you should remove the line

Reply:
I'm going to remove it along with 11940 so that the "real" uniqueVariable cleaning is in the matching PR

Reply:
OK then the pull request is ok to me :)